config/locales/client.en.yml

```diff
@@ -326,7 +326,9 @@ en:
 
   undecided_panel:
     aria_label: 'Undecided members'
-    show_undecided: Show undecided members
+    show: Show
+    remind: Remind
```

Comment:
Hrm. I'd suggest that 'Remind' on its own might not make it very clear what this does. Maybe 'Send Reminder'? Or maybe a button like this with a title?

Here's some thoughts:
![screen shot 2016-12-08 at 4 21 29 am](https://cloud.githubusercontent.com/assets/750477/20996678/770c8200-bcff-11e6-9b1b-d39d2145929b.png)
![screen shot 2016-12-08 at 4 29 37 am](https://cloud.githubusercontent.com/assets/750477/20996673/6c57da30-bcff-11e6-965f-9987b5352c54.png)
![screen shot 2016-12-08 at 4 32 24 am](https://cloud.githubusercontent.com/assets/750477/20996669/6792faca-bcff-11e6-8227-662b9eb21845.png)

@mhjb Do you have thoughts on this?